3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -29,3 +36,5 @@ rollTheDices()
 
 // ! Do not change or remove the code below
 module.export = rollTheDices;
+//Answer: Using Promise all will reject if one of the promises (dices) got out of the table but 
+//all promises have to finish their bending process so they roll even after the reject
```

Comment:
That is true, I will also add that all the promises inside `Promise.all` don't know about each other and executed individually. This is why some dice continue to roll even though other promises were already settled.